setup.py

```diff
@@ -57,6 +57,13 @@ def get_readme() -> str:
     "sphinx-rtd-theme",
     "sphinxcontrib-napoleon",
 ]
+ATARI_REQUIRE = [
+    "opencv-python",
+    "ale-py==0.7.4",
```

Comment:
This is pinned to be exactly that version, any reason why?

Reply:
We rely on getting the render modes of an environment, by accessing the `render.modes` key of the environment's `metadata` dict. In v0.7.5, that key was renamed to `render_modes`. I guess I pinned to 0.7.4 because I thought it would be bad to be too far behind 0.7.5.